packer/builder/azure/driver_restapi/installer/main_test.go

```diff
@@ -3,6 +3,8 @@
 // Licensed under the Apache License, Version 2.0.
 // See License.txt in the project root for license information.
 
+// +build !ci
+
```

Comment:
Hmmm, how do I do that conditionally? Maybe use an environment variable to point to some publishsettings file? If it doesn't exist, skip the test?